src/main/scala/no/ndla/oembedproxy/OEmbedProxyProperties.scala

```diff
@@ -32,10 +32,20 @@ object OEmbedProxyProperties {
     "prod" -> "https://ndla.no/oembed"
   ).getOrElse(Environment, s"https://$Environment.ndla.no/oembed")
 
+  val ListingFrontendOembedServiceUrl = Map(
+    "local" -> "http://localhost:30020/oembed",
+    "prod" -> "https://liste.ndla.no/oembed"
+  ).getOrElse(Environment, s"https://liste.$Environment.ndla.no/oembed")
+
+  val ListingFrontendApprovedUrls = Map(
+    "local" -> List("http://localhost:30020/*"),
+    "prod" -> List("https?://www.liste.ndla.no/*", "https?://liste.ndla.no/*")
+  ).getOrElse(Environment, List(s"https?://www.liste.$Environment.ndla.no/*", s"https?://liste.$Environment.ndla.no/*"))
+
   val NdlaApiOembedProvider = Domain
 
   val NdlaApprovedUrl = Map(
-    "local" -> List("http://api-gateway.ndla-local:30017/*"),
+    "local" -> List("http://localhost:30017/*"),
```

Comment:
skulle denne endringen med?

Reply:
Nei, det er bedre at du setter opp alias i hosts lokalt.

Reply:
Hvis jeg skjønte det riktig så brukes ikke `api-gateway` lenger, så ja det skal nok stemme.

Reply:
ndla-frontend ligger ikke bak api-gateway så dette er riktig.

alternativt ndla-frontend.ndla-local:3000

Reply:
Dette har i praksis ingen betydning, for port 30017 er porten ndla-frontend kjører på om du kjører den i lokal docker. api-gateway.ndla-local er uansett berre et fornuftig alias til localhost å ha i lokal /etc/hosts. Så resultatet er det samme.

Reply:
For meg gir det mer mening at aliaset peker til api-gateway containeren, så hos meg hadde ikke dette fungert.

Når jeg tenker meg om så burde den vel egentlig være `ndla-frontend.ndla-local:3000` for å fungere både i docker og utenfor (med alias i hosts fil).